optaplanner-quarkus-bom/pom.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Where do we place this new module:
1/ optaplanner-quarkus-bom (current location)
2/ build/optaplanner-quarkus-bom
3/ optaplanner-quarkus-integration/optaplanner-quarkus-bom => it's not quarkus-specific, we use it in the `optaplanner-build-parent`

For me it's either 1/ or 2/.